server/src/services/proposal.services.js

```diff
@@ -634,3 +634,17 @@ export const getAllInfoByProposalId = (proposalId, userId) => {
   });
 }
 
+export const archiveExpiredProposals = () => {
+  return new Promise((resolve, reject) => {
+    const now = new Date().toISOString();
+    const sql = 'UPDATE Proposals SET status = "archived" WHERE expiration_date <= ?';
```

Comment:
Shouldn't this be just `<`
Otherwise also the proposal which will be expired at the end of the day are being moved into archived

Reply:
Yes, you are right, I will fix it and send PR again.

Reply:
It's enough to push modifications on the same branch

Reply:
The problem is solved, if there is no other issue, please approve the PR.